src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,360 @@
+
+    {{site.data.alerts.callout_info}}
+    You can also grant the role to a human user. However, during Limited Access, only a service account using the {{ site.data.products.db }} API or Terraform provider can manage folders.
+    {{site.data.alerts.end}}
+
+1. If you grant the role at the level of the organization, it is inherited on all folders. To give your service account the `FOLDER_MOVER` role at the organization level:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://management-staging.crdb.io/api/v1/roles/<service-account-id>/ORGANIZATION/<organization-id>/FOLDER_ADMIN' \
+           --header 'Authorization: Bearer <api-key>'
+    ~~~
+
+    Replace:
+    - `<service-account-id>`: your service account's ID.
+    - `<organization-id>` with the ID you gathered in [Gather Required Information](#gather-required-information).
+    - `<api_key>`: your service account's API key.
+
+To access a folder's clusters, a user or service account must also have the Cluster Administrator, Cluster Creator, or Cluster Operator role on the folder. The role may be granted by inheritance or directly on a cluster.
+
+## List folder contents
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- [Folder Admin](authorization.html#folder-admin) or [Folder Mover](authorization.html#folder-mover)
+
+{{site.data.alerts.callout_success}}
+To discover the entire folder structure, you must list each folder's contents separately.
+{{site.data.alerts.end}}
+
+1. To list the clusters and folders at the level of the organization:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request GET \
+           --url 'https://management-staging.crdb.io/api/v1/folders/root/contents' \
+           --header 'Authorization: Bearer <api_key>'
+    ~~~
+
+    Replace `<api-key>` with your service account's API key.
+
+1. To list the clusters and folders contained within another folder, specify its ID instead of `root`:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request GET \
+           --url 'https://management-staging.crdb.io/api/v1/folders/<folder_id>/contents' \
+           --header 'Authorization: Bearer <api_key>'
+    ~~~
+
+    Replace:
+    - `<folder-id>`: The ID of a folder.
+    - `<api-key>`: Your service account's API key.
+
+{{site.data.alerts.callout_info}}
+In the API, each folder is referred to by its ID, regardless of where it is located in the hierarchy.
+{{site.data.alerts.end}}
+
+## Create a cluster in a folder
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- [Cluster Administrator](authorization.html#cluster-administrator) or [Cluster Creator](authorization.html#cluster-creator)
+- [Folder Admin](authorization.html#folder-admin) or [Folder Mover](authorization.html#folder-mover)
```

Comment:
@mdlinville a user should only need one of the Cluster Roles mentioned above either at the org or the folder, they don't need Folder roles in order to create clusters in a folder.

Reply:
Done